### DISCUSSION TITLE
多租户模式，在基础平台，用户中心，员工维护，对员工重新设置管理部门，在切换企业和组织重新选择单位或部门后，切换，然后退出登录，再次打开切换企业和组织，发现单位和部门为Id，而不是退出前所切换的单位部门

### DISCUSSION THREAD

**Discussion by zwac00001** [zuihou/zuihou-admin-cloud#265] [02.提问]
多租户模式，在基础平台，用户中心，员工维护，对员工重新设置管理部门，在切换企业和组织重新选择单位或部门后，切换，然后退出登录，再次打开切换企业和组织，发现单位和部门为Id，而不是退出前所切换的单位部门

**Comment by zuihou** [comment]
<img width="1562" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/97015165-085e-4f0b-95a9-2740bcb3c9c5">


点击切换按钮时，会调用上面的方法修改 LastCompanyId、LastDeptId 字段。
